src/libraries/Microsoft.Extensions.Options/Microsoft.Extensions.Options.sln

```diff
@@ -65,6 +65,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Hostin
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Hosting", "..\Microsoft.Extensions.Hosting\src\Microsoft.Extensions.Hosting.csproj", "{6AD51705-102E-4E9D-B62E-6BF19025F3A7}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Logging.Generators", "..\Microsoft.Extensions.Logging.Abstractions\gen\Microsoft.Extensions.Logging.Generators.Roslyn3.11.csproj", "{E00A72D2-A6C2-4334-92AC-7CF7ECC87F4D}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Logging.Generators", "..\Microsoft.Extensions.Logging.Abstractions\gen\Microsoft.Extensions.Logging.Generators.Roslyn4.0.csproj", "{EBEF8790-8095-4FD9-8436-1354B30E1529}"
+EndProject
```

Comment:
I built the `slngen.proj` project in 27034bc84b3ffb7f311425fee03ebb24fee55e82 to regenerate all of the solutions since I added new project references for the DllImportGenerator. Looks like someone didn't run it beforehand and this update also updated the solutions with the logging source generator as well.